golang/x/relay/keeper/request_test.go

```diff
@@ -165,7 +165,8 @@ func (s *KeeperSuite) TestCheckRequests() {
 
 	// Success
 	in := btcspv.ExtractInputAtIndex(v.Vin, v.InputIdx)
-	requestErr = s.Keeper.setRequest(s.Context, in, out[8:], 10, 255)
+	outpoint := btcspv.ExtractOutpoint(in)
+	requestErr = s.Keeper.setRequest(s.Context, outpoint, out[8:], 10, 255)
```

Comment:
cool. for future readers, can we make a note anywhere the `[8:]` is seen on an output? basically a note that says we're extracting the output script